src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" 
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+          href="https://fonts.gstatic.com" 
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <img class="imac_photo"
+           src="images/imac.jpeg" 
+           alt="imac">
+      <div class="product">
+        <p class="product_name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="product_code">Product code: 195434</p>
+      </div>
+      <div class="container__card card">
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/199043419-a80d975c-550a-48d8-88d2-4dbdf4bd2d8c.png)

And use a separate file for stars block